payjoin/src/receive/error.rs

```diff
@@ -45,6 +45,13 @@ impl From<crate::ohttp::OhttpEncapsulationError> for Error {
     fn from(e: crate::ohttp::OhttpEncapsulationError) -> Self { Error::Server(Box::new(e)) }
 }
 
+#[derive(Debug)]
+pub struct PsbtInputError(crate::psbt::InternalPsbtInputError);
```

Comment:
Did you mean to define this public error type twice? It's defined in both `psbt.rs` and `receive/error.rs`.

Reply:
fixed now